docker/docker-compose.yml

```diff
@@ -21,36 +21,36 @@ services:
       - "8010:8080"
     volumes:
       - girder_assetstore:/home/assetstore
-      - pipelines:${VIAME_PIPELINES_PATH:-/home/VIAME/pipelines}:ro
+      # Mount pipeline directory from girder_worker
+      - pipelines:${VIAME_PIPELINES_PATH}:ro
       - type: bind
         source: ${PUBLIC_DATA_PATH}
         target: /data/public
         read_only: true
         volume:
           nocopy: true
     environment:
-      - VIAME_PIPELINES_PATH=${VIAME_PIPELINES_PATH:-/home/VIAME/pipelines}
+      - VIAME_PIPELINES_PATH=${VIAME_PIPELINES_PATH}
       - GIRDER_MONGO_URI=mongodb://mongo:27017/girder
       - GIRDER_ADMIN_USER=${GIRDER_ADMIN_USER}
       - GIRDER_ADMIN_PASS=${GIRDER_ADMIN_PASS}
+      - CELERY_BROKER_URL=amqp://guest:guest@rabbit/
+      - BROKER_CONNECTION_TIMEOUT=2
 
   girder_worker:
+    runtime: 'nvidia'
+    build:
+      context: ../
+      dockerfile: docker/girder_worker.Dockerfile
     volumes:
-      - ./server:/home/viame_girder
-      - pipelines:/home/VIAME/build/install/configs/pipelines
+      # Exposes pipeline directory to girder so it can see what pipelines are available.
+      - pipelines:${VIAME_PIPELINES_PATH}:ro
     depends_on:
       - rabbit
     environment:
-      - GIRDER_WORKER_BROKER=amqp://guest:guest@rabbit/
-      - GIRDER_WORKER_BACKEND=amqp://guest:guest@rabbit/
-      - GIRDER_WORKER_PLUGINS_ENABLED=girder_io
+      - VIAME_PIPELINES_PATH=${VIAME_PIPELINES_PATH}
```

Comment:
Small thing, but I don't think this needs to be here, as it's only used by girder and not girder worker.

Reply:
See https://github.com/VIAME/VIAME-Web/pull/77/commits/43ef72f6e22d7e3c807c7be1d400e064c087b700#diff-46d897de59fd6accaec892ee86d50f15L12

Reply:
Oops, my bad.